### ISSUE TITLE
im cannot open file from JD-GUI...

### ISSUE THREAD

**Issue by Glaezz** [java-decompiler/jd-gui#335]
I will insert the file into the JD-GUI, using the open file feature, but when I click the open file button, I can't find the file selection pop up, nor does the drag and drop feature work.

windows 7/64 Bit

**Comment by Code4SAFrankie** [comment]
Also can't open or drag open class files. Does not give any error or list in the recently used files list.

**Comment by konstantinj** [comment]
Same here. Was working some weeks ago, now not anymore. I'm on OSX, guess some brew update or so broke it.

**Comment by adobni** [comment]
I can't open anything from Downloads, but I can drag and drop from Downloads into JD-GUI. This happened from several months ago, since the upgrade to Catalina last year I think.

**Comment by kumar1210** [comment]
I am not able to open any class file from mac, drag-drop doesn't open. And navigating to the particular folder doesn't even show the class files.

**Comment by zerog2k** [comment]
At least for me with Mac Catalina, I found that there is something about security policy which prohibits jd-gui from accessing certain directory structures, like ~/Documents for example. It will not see any directory structure beyond it. If I copy the file structure I want to examine into another accessible dir, like `/tmp/` then I am able to browse.

I thought it might have something to do with either the xattr com.apple.quarantine, but I think I have removed it from both jd-gui as well as the target files w/ `/usr/bin/xattr -d com.apple.quarantine FILENAME`, but still didn't work.

**Comment by jjangga0214** [comment]
I installed openjdk 11 through homebrew on macOS, but jd-gui could not detect it.

Thus I add this line at the top of the file.

And this made it work.

**Contents/MacOS/universalJavaApplicationStub.sh** 
(In the "JD-GUI.app" package -  Finder -> App Icon -> Right click -> Look into package )

```sh
export JAVA_HOME="/opt/homebrew/opt/openjdk@11"
```
